packages/block-library/src/template-part/editor.scss

```diff
@@ -79,10 +78,6 @@
 			margin-right: 8px;
 		}
 	}
-
-	.wp-block-template-part__preview-dropdown-button {
-		border-right: $border-width solid $gray-900;
-	}
```

Comment:
I've updated the code to use `ToolbarGroup` instead of just a `<div>`, so these styles that were simulating a toolbar group aren't needed anymore.